### DISCUSSION TITLE
Translate invite

### DISCUSSION THREAD

**Discussion by shirou93** [immich-app/immich#736] [Q&A]
Hi, can i got invite to translate english to polish?

Actual version mobile app have partly translate.

**Comment by alextran1502** [comment]
Can you send me your email address?

**Reply by shirou93** [reply]
bartek04041993@gmail.com

**Comment by owmsoton** [comment]
a few days before, I did a translation for pt_BR, but I did it as a PR.

here my email **oms.moreira@outlook.com**

invite me too.
